classes/ActionScheduler_wpPostStore.php

```diff
@@ -207,7 +261,8 @@ public function query_actions( $query = array() ) {
 
 		/** @var wpdb $wpdb */
 		global $wpdb;
-		$sql = "SELECT p.ID FROM {$wpdb->posts} p";
+		$sql  = ( 'count' === $select_or_count ) ? 'SELECT count(p.ID)' : 'SELECT p.ID ';
```

Comment:
I think it would be a good idea to validate the value of the `$select_or_count` variable, since there are only 2 valid values.